src/hotspot/share/classfile/stringTable.cpp

```diff
@@ -650,6 +654,32 @@ size_t StringTable::verify_and_compare_entries() {
   return vcs._errors;
 }
 
+static void print_string(Thread* thr, outputStream* st, oop s) {
+  typeArrayOop value     = java_lang_String::value_no_keepalive(s);
+  int          length    = java_lang_String::length(s);
+  bool         is_latin1 = java_lang_String::is_latin1(s);
```

Comment:
Nit: there is no need to align things this way.

Reply:
I'll leave it this time since the code was factored out from existing code.

Thanks for your review.